tests/cnf/ran/oran/tests/oran-provision.go

```diff
@@ -1 +1,134 @@
 package tests
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/configmap"
+	"github.com/openshift-kni/eco-goinfra/pkg/ocm"
+	"github.com/openshift-kni/eco-goinfra/pkg/oran"
+	"github.com/openshift-kni/eco-goinfra/pkg/reportxml"
+	"github.com/openshift-kni/eco-goinfra/pkg/secret"
+	"github.com/openshift-kni/eco-goinfra/pkg/siteconfig"
+	. "github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/raninittools"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/oran/internal/helper"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/oran/internal/tsparams"
+	policiesv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// ContinueOnError is deliberately left out of this Ordered container. If the invalid ProvisioningRequest does not
+// become valid again, we cannot test provisioning with a valid ProvisioningRequest.
+var _ = Describe("ORAN Provision Tests", Label(tsparams.LabelProvision), Ordered, func() {
+	var prBuilder *oran.ProvisioningRequestBuilder
+
+	AfterAll(func() {
+		By("verifying the spoke 1 kubeconfig path is set")
+		Expect(RANConfig.Spoke1Kubeconfig).
+			ToNot(BeEmpty(), "KUBECONFIG environment variable must be set to save spoke 1 kubeconfig")
+
+		By("saving the spoke 1 admin kubeconfig")
```

Comment:
Is this only needed after successful deployment? If so, this should be in AfterEach of that specific test case.